<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>850</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>850</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000</width>
    <height>850</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>DAMOA</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url(:/img/DAMOA 500.png);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>230</y>
      <width>500</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>맑은 고딕</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::DefaultContextMenu</enum>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color : rgb(255, 255, 255, 110);
color : #000000;
border : 0px</string>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Normal</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="placeholderText">
     <string>원하는 단어를 입력해주세요.</string>
    </property>
    <property name="cursorMoveStyle">
     <enum>Qt::LogicalMoveStyle</enum>
    </property>
    <property name="clearButtonEnabled">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QDateEdit" name="sDate">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>310</y>
      <width>95</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>맑은 고딕</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">
</string>
    </property>
    <property name="dateTime">
     <datetime>
      <hour>0</hour>
      <minute>0</minute>
      <second>0</second>
      <year>2019</year>
      <month>1</month>
      <day>1</day>
     </datetime>
    </property>
    <property name="minimumDateTime">
     <datetime>
      <hour>0</hour>
      <minute>0</minute>
      <second>0</second>
      <year>1990</year>
      <month>1</month>
      <day>1</day>
     </datetime>
    </property>
    <property name="maximumDate">
     <date>
      <year>2100</year>
      <month>12</month>
      <day>31</day>
     </date>
    </property>
    <property name="minimumDate">
     <date>
      <year>1990</year>
      <month>1</month>
      <day>1</day>
     </date>
    </property>
    <property name="calendarPopup">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QDateEdit" name="eDate">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>310</y>
      <width>95</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>맑은 고딕</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">
</string>
    </property>
    <property name="dateTime">
     <datetime>
      <hour>0</hour>
      <minute>0</minute>
      <second>0</second>
      <year>2019</year>
      <month>9</month>
      <day>17</day>
     </datetime>
    </property>
    <property name="maximumDate">
     <date>
      <year>7999</year>
      <month>12</month>
      <day>31</day>
     </date>
    </property>
    <property name="minimumDate">
     <date>
      <year>1990</year>
      <month>1</month>
      <day>1</day>
     </date>
    </property>
    <property name="calendarPopup">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>310</y>
      <width>90</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>맑은 고딕</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255)</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;&lt;span style=&quot; font-size:12pt; color:#000000;&quot;&gt;시작 날짜 : &lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>310</y>
      <width>90</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>맑은 고딕</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255)</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;&lt;span style=&quot; font-size:12pt; color:#000000;&quot;&gt;끝 날짜 : &lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="pBar">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>800</y>
      <width>600</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>맑은 고딕</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color : rgb(222, 237, 249, 60);
color:#000000</string>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>800</y>
      <width>70</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>맑은 고딕</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255)</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;&lt;span style=&quot; font-size:12pt; color:#000000;&quot;&gt;진행률 : &lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Search">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>230</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>맑은 고딕</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color : rgb(21, 37, 55, 0);
color:#ffffff;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>img/Search.png</normaloff>img/Search.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>40</width>
      <height>40</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="Process">
    <property name="geometry">
     <rect>
      <x>825</x>
      <y>800</y>
      <width>75</width>
      <height>26</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>맑은 고딕</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::DefaultContextMenu</enum>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color : rgb(172, 200, 233);
color:#505050;</string>
    </property>
    <property name="text">
     <string>가공</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <property name="autoRepeat">
     <bool>false</bool>
    </property>
    <property name="autoExclusive">
     <bool>false</bool>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QComboBox" name="cb_ctgr">
    <property name="geometry">
     <rect>
      <x>625</x>
      <y>310</y>
      <width>100</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>맑은 고딕</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color : rgb(255, 255, 255)</string>
    </property>
    <item>
     <property name="text">
      <string>선택 안함</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>정치</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>경제</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>사회</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>생활/문화</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>세계</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>IT/과학</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="cb_ss">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>310</y>
      <width>100</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>맑은 고딕</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color : rgb(255, 255, 255)</string>
    </property>
    <property name="maxVisibleItems">
     <number>12</number>
    </property>
    <property name="minimumContentsLength">
     <number>0</number>
    </property>
    <item>
     <property name="text">
      <string>선택 안함</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>535</x>
      <y>310</y>
      <width>90</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>맑은 고딕</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255)</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;&lt;span style=&quot; font-size:12pt; color:#000000;&quot;&gt;카테고리 : &lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>310</y>
      <width>90</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>맑은 고딕</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255)</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;&lt;span style=&quot; font-size:12pt; color:#000000;&quot;&gt;섹션 : &lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>350</y>
      <width>800</width>
      <height>420</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color : rgb(50)</string>
    </property>
    <property name="tabPosition">
     <enum>QTabWidget::North</enum>
    </property>
    <property name="tabShape">
     <enum>QTabWidget::Rounded</enum>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <property name="elideMode">
     <enum>Qt::ElideNone</enum>
    </property>
    <property name="documentMode">
     <bool>true</bool>
    </property>
    <property name="tabsClosable">
     <bool>false</bool>
    </property>
    <property name="movable">
     <bool>false</bool>
    </property>
    <property name="tabBarAutoHide">
     <bool>false</bool>
    </property>
    <widget class="QWidget" name="tab_1">
     <attribute name="title">
      <string>수집 내용</string>
     </attribute>
     <widget class="QTextEdit" name="textEdit">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>800</width>
        <height>400</height>
       </rect>
      </property>
      <property name="tabletTracking">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color : rgb(255, 255, 255, 110)</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Gulim'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>통계 그래프</string>
     </attribute>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>800</width>
        <height>400</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color : rgb(255, 255, 255, 110)</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>워드 클라우드</string>
     </attribute>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>800</width>
        <height>400</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color : rgb(255, 255, 255, 110)</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1000</width>
      <height>850</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color : rgb(255, 255, 255)</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>img/Background1_4.png</pixmap>
    </property>
   </widget>
   <zorder>label_9</zorder>
   <zorder>label_2</zorder>
   <zorder>label_3</zorder>
   <zorder>pBar</zorder>
   <zorder>label_4</zorder>
   <zorder>Process</zorder>
   <zorder>label_6</zorder>
   <zorder>label_7</zorder>
   <zorder>sDate</zorder>
   <zorder>eDate</zorder>
   <zorder>cb_ctgr</zorder>
   <zorder>cb_ss</zorder>
   <zorder>lineEdit</zorder>
   <zorder>tabWidget</zorder>
   <zorder>Search</zorder>
  </widget>
 </widget>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>lineEdit</tabstop>
  <tabstop>sDate</tabstop>
  <tabstop>eDate</tabstop>
  <tabstop>cb_ctgr</tabstop>
  <tabstop>cb_ss</tabstop>
  <tabstop>Search</tabstop>
  <tabstop>Process</tabstop>
  <tabstop>textEdit</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
